packages/api/src/models/animalBreedModel.js

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import baseModel from './baseModel.js';
+
+class AnimalBreed extends baseModel {
+  static get tableName() {
+    return 'animal_breed';
+  }
+
+  static get idColumn() {
+    return 'id';
+  }
+
+  // Optional JSON schema. This is not the database schema! Nothing is generated
+  // based on this. This is only used for validation. Whenever a model instance
+  // is created it is checked against this schema. http://json-schema.org/.
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['farm_id', 'type_id', 'breed'],
+
+      properties: {
+        id: { type: 'integer' },
+        farm_id: { type: 'string' },
+        type_id: { type: 'integer' },
+        breed: { type: 'string' },
+        // breed_key will not be provided for user-generated types and should default to null
```

Comment:
Even if null should be added below as part of the schema.

Reply:
That's my bad and I'll change it in the other one too! I thought setting it to `null` would force having to pass an explicit `null` from the controller, but that is not the case.